cli/src/bin/ulid.rs

```diff
@@ -29,11 +30,24 @@ fn main() {
 
 fn generate(count: u32, monotonic: bool) {
     let stdout = io::stdout();
+    let stderr = io::stderr();
     let mut locked = stdout.lock();
+    let mut err_locked = stderr.lock();
     if monotonic {
         let mut gen = Generator::new();
-        for _ in 0..count {
-            writeln!(&mut locked, "{}", gen.generate().unwrap()).unwrap();
+        let mut i = 0;
+        while i < count {
+            match gen.generate() {
+                Ok(ulid) => {
+                    writeln!(&mut locked, "{}", ulid).unwrap();
+                    i += 1;
+                }
+                Err(_) => {
+                    writeln!(&mut err_locked, "unable to create ulid").unwrap();
```

Comment:
Can this be something with more explanation like "Failed to create new ulid due to overflow, sleeping 1 ms"